src/main/java/nextstep/ladder/model/Line.java

```diff
@@ -0,0 +1,93 @@
+package nextstep.ladder.model;
+
+import nextstep.ladder.exception.MinimumException;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.function.IntFunction;
+import java.util.function.UnaryOperator;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public final class Line {
+
+    private static final int ZERO = 0;
+    private static final int ONE = 1;
+    private static final boolean FALSE = false;
+    private static final String MINIMUM_LINE_MESSAGE = "사다리 높이는 최소 2개 이상이어야 합니다.";
+    private static final String WIDTH_LINE = "-----";
+    private static final String EMPTY_WIDTH_LINE = "     ";
+    private static final String HEIGHT_LINE = "|";
+    private static final String EMPTY_HEIGHT_LINE = "";
+
+    private final List<Boolean> points;
+
+    private Line(List<Boolean> points) {
+        this.points = points;
+    }
+
+    private static void check(int countOfPeople) {
+        Optional.of(countOfPeople)
+                .filter(count -> 2 < count)
+                .orElseThrow(() -> new MinimumException(MINIMUM_LINE_MESSAGE));
+    }
+
+    public static Line create(int countOfPeople) {
+        check(countOfPeople);
+        List<Boolean> points = init()
+                .andThen(removable())
+                .apply(countOfPeople);
+        return new Line(points);
+    }
+
+    private static Function<Integer, List<Boolean>> init() {
+        return countOfPeople -> IntStream
+                .range(ZERO, countOfPeople)
+                .mapToObj(i -> Math.round(Math.random()) == ONE)
+                .collect(Collectors.toList());
+    }
+
+    private static UnaryOperator<List<Boolean>> removable() {
+        return points -> {
+            int lastIndex = points.size() - ONE;
+            IntStream.range(ZERO, lastIndex)
+                    .filter(i -> points.get(i) && points.get(i + ONE))
+                    .forEach(i -> points.set(i + ONE, FALSE));
```

Comment:
해당 두 라인은 각각 어떤 기능을 수행하는지 직관적으로 파악하기가 어려운 코드로 보여요.
각각을 메서드로 분리하고 이름을 부여함으로써 가독성을 높여볼 수 있지 않을까 싶은데, 어덯게 생각하시나요?
이름은 단순한 예시이니 참고만 부탁드릴게요.
```suggestion
                    .filter(i -> doAdjacentPointsHaveLines(points, i))
                    .forEach(i -> setNextPointNoLine(points, i));
```